docs/commands.md

```
@@ -334,6 +335,122 @@ AI:  Verifying add-dark-mode...
 
+### `/opsx:clarify`
+
+Clarify underspecified areas in an OpenSpec change by identifying ambiguities and resolving them through interactive Q&A. Writes answers directly back into spec artifacts.
+
+**Syntax:**
+```
+/opsx:clarify [change-name]
+```
+
+**Arguments:**
+| Argument | Required | Description |
```

markdownlint-cli2 (0.20.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)